From 35fd62ebb87a7d6538529c3edb2e2acc4724af83 Mon Sep 17 00:00:00 2001
From: Cedric Bondier <cedric.bondier@intel.com>
Date: Tue, 20 Jan 2015 16:46:43 +0100
Subject: [PATCH] [BT] remove rfkill-gpio handling for BT chip.

rfkill gpio handling will be handled by bt_lpm driver.

Change-Id: Ib46e19382de65e6e3a13415facfdb24c0454a107
Tracked-On: https://jira01.devtools.intel.com/browse/GMINL-7915
Signed-off-by: Cedric Bondier <cedric.bondier@intel.com>
---
 net/rfkill/rfkill-gpio.c | 22 ----------------------
 1 file changed, 22 deletions(-)

diff --git a/net/rfkill/rfkill-gpio.c b/net/rfkill/rfkill-gpio.c
index 321fe9fa5c03..e00641da15e4 100644
--- a/net/rfkill/rfkill-gpio.c
+++ b/net/rfkill/rfkill-gpio.c
@@ -303,14 +303,6 @@ static int rfkill_gpio_remove(struct platform_device *pdev)
 }
 
 #ifdef CONFIG_ACPI
-static struct rfkill_gpio_desc acpi_default_bluetooth = {
-	.type = RFKILL_TYPE_BLUETOOTH,
-	.reset_idx = 0,
-	.shutdown_idx = 1,
-	.wake_idx = -1,
-	.host_wake_idx = -1,
-};
-
 static struct rfkill_gpio_desc acpi_default_gps = {
 	.type = RFKILL_TYPE_GPS,
 	.reset_idx = 0,
@@ -319,15 +311,6 @@ static struct rfkill_gpio_desc acpi_default_gps = {
 	.host_wake_idx = -1,
 };
 
-static struct rfkill_gpio_desc acpi_bluetooth_wake = {
-	.type = RFKILL_TYPE_BLUETOOTH,
-	.reset_idx = -1,
-	.shutdown_idx = 1,
-	.wake_idx = 0,
-	.host_wake_idx = 2,
-	.host_wake_trigger = IRQF_TRIGGER_RISING,
-};
-
 static struct rfkill_gpio_desc acpi_gps_wake = {
 	.type = RFKILL_TYPE_GPS,
 	.reset_idx = -1,
@@ -347,11 +330,6 @@ static struct rfkill_gpio_desc acpi_gps_wake_falling = {
 };
 
 static const struct acpi_device_id rfkill_acpi_match[] = {
-	{ "BCM2E1A", (kernel_ulong_t)&acpi_default_bluetooth },
-	{ "BCM2E39", (kernel_ulong_t)&acpi_default_bluetooth },
-	{ "BCM2E3D", (kernel_ulong_t)&acpi_default_bluetooth },
-	{ "BCM2E3A", (kernel_ulong_t)&acpi_bluetooth_wake },
-	{ "OBDA8723", (kernel_ulong_t)&acpi_bluetooth_wake },
 	{ "BCM4752", (kernel_ulong_t)&acpi_default_gps },
 	{ "LNV4752", (kernel_ulong_t)&acpi_default_gps },
 	{ "BCM4752E", (kernel_ulong_t)&acpi_default_gps },
-- 
1.9.1

